src/status_im/utils/signing_phrase/core.cljs

```diff
@@ -3,6 +3,12 @@
             [status-im.utils.signing-phrase.dictionaries.ru :as ru]
             [clojure.string :as string]))
 
+;`signing-phrase` is a language-specific combination of three 4-lettered words
```

Comment:
Could you elaborate a bit more here on why and how it is used? When you first get prompted with it, what attack it guards against, how you use it in transactions etc

Reply:
The workflow isn't fully defined yet. The scope of this issue and PR is more or less just to generate the phrase and store it. I had to expand the scope to cover the migration from previous db version and account recovery simply because the app could've gotten in an inconsistent state otherwise.

Reply:
The best I can do is possibly reword/retype the description of the original issue: https://github.com/status-im/status-react/issues/1585

Reply:
That's fine, you can copy paste issue description if it accurately describes goal of code. Main thing is to capture it for the future so we don't have to go hunting in issue history